### ISSUE TITLE
Fails for boolean attributes

### ISSUE THREAD

**Issue by des-des** [rayd/html-parse-stringify2#13]
@rayd 
I am getting a problem in a dependency (snabbdom-virtualize). I think it is being caused by the way this lib handles boolean attributes.

like
```html
<script async type='text/javascript' ...
```
when this gets parsed we end up with an attribute name `text/javascript` which causes an error when I update the dom.

**Comment by des-des** [comment]
Yeah, here
https://github.com/rayd/html-parse-stringify2/blob/master/lib/parse-tag.js#L16
I think you are assuming that the attributes all have values.

Although the script example is a bit weird, I image this would cause problems with
```html
<input checked ...
```
which is valid html

**Comment by des-des** [comment]
Here is the failing test case
```js

    tag = '<input checked name="sad" type="checkbox">';

    t.deepEqual(parseTag(tag), {
        type: 'tag',
        attrs: {
            type: 'checkbox',
            checked: 'checked', // This conforms to spec for boolean attributes
            name: 'sad'
        },
        name: 'input',
        voidElement: false,
        children: []
    });
```

With output
```
 expected:
      { attrs: { checked: 'checked', name: 'sad', type: 'checkbox' }, children: [], name: 'input', type: 'tag', voidElement: false }
    actual:
      { attrs: { '"sad"': 'type', checked: 'name' }, children: [], name: 'input', type: 'tag', voidElement: true }

```

**Comment by rayd** [comment]
Fixed in 2.0.1 with #14
